contracts/extensions/votingReputation/VotingReputationStaking.sol

```diff
@@ -49,7 +49,10 @@ contract VotingReputationStaking is VotingReputationStorage {
     // For v9 expenditure motions, only allow counterstaking unless escalated
     if (
       _motionId <= motionCountV10 &&
-      isExpenditureSig(getActionSummary(motion.action, motion.altTarget).sig)
+      isExpenditureSig(
+        getActionSummary(address(colonyNetwork), address(colony), motion.action, motion.altTarget)
```

Comment:
I'm noticing that every invocation casts to address, and then re-casts inside of `getActionSummary`. What do you think about passing in `type IColonyNetwork`, etc directly?